elephant/conversion.py

```diff
@@ -152,3 +155,704 @@ def binarize(spiketrain, sampling_rate=None, t_start=None, t_stop=None,
+
+    @property
+    def edges(self):
+        """
+        Returns all time edges with :attr:`num_bins` bins as a quantity array.
+
+        The borders of all time steps between start and stop [start, stop]
+        with:attr:`num_bins` bins are regarded as edges.
+        The border of the last bin is included.
+
+        Returns
+        -------
+        bin_edges : quantities.Quantity array
+            All edges in interval [start, stop] with :attr:`num_bins` bins
+            are returned as a quantity array.
+        """
+        return np.linspace(self.t_start, self.t_stop,
+                           self.num_bins + 1, endpoint=True)
+
+    @property
+    def left_edges(self):
+        """
+        Returns an quantity array containing all left edges and
+        :attr:`num_bins` bins.
+
+        The left borders of all time steps between start and excluding stop
+        [start, stop) with number of bins from given input are regarded as
+        edges.
+        The border of the last bin is excluded.
+
+        Returns
+        -------
+        bin_edges : quantities.Quantity array
+            All edges in interval [start, stop) with :attr:`num_bins` bins
+            are returned as a quantity array.
+        """
+        return np.linspace(self.t_start, self.t_stop, self.num_bins,
+                           endpoint=False)
+
+    @property
+    def right_edges(self):
+        """
+        Returns the right edges with :attr:`num_bins` bins as a
+        quantities array.
+
+        The right borders of all time steps between excluding start and
+        including stop (start, stop] with :attr:`num_bins` from given input
+        are regarded as edges.
+        The border of the first bin is excluded, but the last border is
+        included.
+
+        Returns
+        -------
+        bin_edges : quantities.Quantity array
+            All edges in interval [start, stop) with :attr:`num_bins` bins
+            are returned as a quantity array.
+        """
+        return self.left_edges + self.binsize
+
+    @property
+    def center_edges(self):
+        """
+        Returns each center time point of all bins between start and stop
+        points.
+
+        The center of each bin of all time steps between start and stop
+        (start, stop).
+
+        Returns
+        -------
+        bin_edges : quantities.Quantity array
+            All center edges in interval (start, stop) are returned as
+            a quantity array.
+        """
+        return self.left_edges + self.binsize / 2
+
+    @property
+    def sparse_mat_unclip(self):
+        """
+        Getter for **unclipped** version of the sparse matrix.
+
+        Returns
+        -------
+        matrix: scipy.sparse.csr_matrix
+            Sparse matrix, counted, unclipped version.
+
+        See also
+        --------
+        scipy.sparse.csr_matrix
+        mat_unclipped
+        """
+        if self._sparse_mat_u is not None:
```

Comment:
If this is `None`, then the property will return `None`.  Is this what you really want?  If so, it should be documented.